api/storageprovisioner/provisioner.go

```diff
@@ -243,6 +243,26 @@ func (st *State) VolumeParams(tags []names.VolumeTag) ([]params.VolumeParamsResu
 	return results.Results, nil
 }
 
+// DestroyVolumeParams returns the parameters for destroying the volumes
+// with the specified tags.
+func (st *State) DestroyVolumeParams(tags []names.VolumeTag) ([]params.DestroyVolumeParamsResult, error) {
```

Comment:
Renamed to RemoveVolumeParams (same for Filesystem), and changed the field from Release to Destroy.